### PR TITLE
Fixing UX bugs from #126

### LINKED ISSUE #126: Bug — UX Tweaks
**Describe the bug**
Collection of UX tweaks we should add to make YACS.N ready for v0 release.
Given there are a few, will not make an issue for each. Tick the checkmark for each when each is done.

- [x] Click the class to add it to selected courses
> Screen Shot 2020-04-24 at 12 26 29 PM
- [x] Click the header of the class to open/close in "selected courses"
> Screen Shot 2020-04-24 at 12 10 49 PM
- [x] Downwards tick should change to upwards tick when opened/closed in "Selected Courses"
> Screen Shot 2020-04-24 at 12 13 24 PM
- [x] Show that no courses are selected, please add courses from the search
> Screen Shot 2020-04-24 at 12 13 51 PM
- [x] "Other semester" buttons should be text, either bold or greyed out if not-selected, currently it draws too much attention
Screen Shot 2020-04-24 at 12 24 17 PM

### PR DESCRIPTION
**Pull request by marchdan**
**Issue**
closes #126 

**Test Procedure**

Will be updated as features are fixed:

> 1. Load public semesters through admin page
> 2. On main page, look at footer and see that the other semesters are black, bolded text
> 3. Click on the title of a course in Course search, and make sure it gets added to Selected courses
> 4. Go to selected courses and click on the title of the course, the sections should collapse and the chevron should switch from pointing up to pointing down.
> 5. Remove all courses from selected courses, a warning should appear telling you to add courses
> 6. Leave a comment with thoughts about the warning and how it should be improved. 

**Photos**
`Before:`
other-semesters-before
`After:`
other-semesters

`Before:`
no-course-before
`After:`
no-course-after

### CODE REVIEW

File: src/web/src/components/CourseListing.vue [hunk 1]
```diff
@@ -1,7 +1,14 @@
 <template>
   <div>
     <div class="d-flex w-100 justify-content-between">
-      <div>
+      <div v-if="actions.add" @click="toggleCourse(course)">
```
Comment (123joshuawu) [inline src/web/src/components/CourseListing.vue:4]: what about `@click="actions.add ? toggleCourse(course) : toggleShowSection()"`
or move it to another function like `defaultToggleAction`

File: src/web/src/components/CourseListing.vue [hunk 2]
```diff
@@ -1,7 +1,14 @@
 <template>
   <div>
     <div class="d-flex w-100 justify-content-between">
```
Comment (123joshuawu) [inline src/web/src/components/CourseListing.vue:3]: we probably also want `cursor:pointer` so ppl know its clickable

**Comment by jshom** [comment]
Too much margin, and little too large
- [ ] reduce size 
- [ ] reduce margin
Screen Shot 2020-04-24 at 5 51 06 PM

**Comment by jshom** [comment]
With the no results in "Selected Course's"
lets change the style on border color and enlarge padding a bit
Screen Shot 2020-04-24 at 5 54 10 PM
Screen Shot 2020-04-24 at 5 57 23 PM

**Comment by jshom** [comment]
We should be able to click on the body, not just the course title to get select the course.

Kapture 2020-04-24 at 18 08 14

**Comment by marchdan** [comment]
> Too much margin, and little too large
> 
> * [x]  reduce size
> * [x]  reduce margin
> 
> Style should conform to the rest of the footer
> 
> Screen Shot 2020-04-24 at 5 51 06 PM

So no bold??

**Comment by marchdan** [comment]
@jshom Please review so we can merge

**Pull request merged**